### ISSUE TITLE
mouse:up event in Safari?

### ISSUE THREAD

**Issue by lucidchin** [altert/OpenseadragonFabricjsOverlay#26]
I'm seeing something odd. It looks like in Safari, the mouse:up event for the fabric overlay doesn't get triggered. Anyone else run into Safari issues?

**Comment by altert** [comment]
does it happen on current demo? http://altert.github.io/OpenseadragonFabricjsOverlay/demo.html

**Comment by lucidchin** [comment]
Hmm, should I be using the fabric.adapted.js in the demo? In my project I'm using fabric-1.7.7.js.
I modified the openseadragon-fabricjs-overlay.js file and just added the following code:

```
this._fabricCanvas.on('mouse:up',function(options) {
     console.log("Mouse Up");
});
```

But the Mouse Up message never appears in the debug console. I put a similar console.log in the mouse:down and that appears in the console.

**Comment by altert** [comment]
yes, see https://github.com/altert/OpenseadragonFabricjsOverlay/pull/21
My guess it's similar. Demo works fine for me in safari

**Comment by lucidchin** [comment]
Thanks for the info. I finally got things working by using the updated files from the demo. But something I noticed is the mouse:up event doesn't fire unless the openseadragon viewer has the mouseNavEnabled set to false. The mouse:down and mouse:move are still triggered though.
I also didn't realize that I had to have the e.preventDefault in the mouse:down, without that the mouse:up also didn't trigger.
